.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 *.png
+*.DS_Store
```

Comment:
add EOL

Reply:
.gitignore file is special , when you add blank line ,  git will not detect such change. you can try it locally .